### ISSUE TITLE
The highlight element was blocked and z-index was invalid while the position style of parentNode is fixed or sticky.

### ISSUE THREAD

**Issue by nicoopy** [kamranahmedse/driver.js#332]
Could we do this to solve the problem ? (Set position relative for these parentNodes of the highlight element)
At /src/core/element.js:
```
fixStackingContext() {
  // ...
  const position = getStyleProperty(parentNode, 'position', true);
  if (position === 'sticky' || position === 'fixed') {
    parentNode.classList.add(CLASS_POSITION_RELATIVE);
  }
  // ...
}

removeHighlightClasses() {
  // ...
  let parentNode = this.node.parentNode;
  while (parentNode) {
    if (!parentNode.tagName || parentNode.tagName.toLowerCase() === 'body') {
      break;
    }
    if (parentNode.classList.value.indexOf(CLASS_POSITION_RELATIVE) > -1) {
      parentNode.classList.remove(CLASS_POSITION_RELATIVE);
    }
    parentNode = parentNode.parentNode;
  }
  // ...
}
```

**Comment by AMITSINGH1** [comment]
const driver = new Driver({
            opacity: 0.75, 
            padding: 0, 
            allowClose: false, 
            animate: false,
            showButtons: true,
          });

**Comment by josias-r** [comment]
I ran into a zIndex issue too, and the `fixStackingContext` hacks were not applicable to my issues. Therefore I [ported driver.js and refactored it](https://josias-r.github.io/boarding.js/) to have it work with an alternative concept that is more stable and does not reply on an zIndex logic.

This library would mostlikely avoid running into the issues you are having

**Comment by kamranahmedse** [comment]
We just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. This issue should no longer exist. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
